src/react-sortable-tree.js

```diff
@@ -361,7 +363,7 @@ class ReactSortableTree extends Component {
     }
 
     renderRow({ node, path, lowerSiblingCounts, treeIndex }, listIndex, key, style, getPrevRow, matchKeys) {
-        const NodeContentRenderer = this.nodeContentRenderer;
+        const NodeContentRenderer = this.state.nodeContentRenderer || this.props.nodeContentRenderer;
```

Comment:
When you use `this.props.nodeContentRenderer` directly, it won't have the props related to drag-and-drop injected into it. It needs to be wrapped with `dndWrapSource`, as in the constructor.

I made a decision when writing this that the `nodeContentRenderer` would only be referenced once, at component mount time. As it won't respond to changes, I kept it out of `this.state`.